detox/e2e/support/server_api/default_config.json

```diff
@@ -0,0 +1,296 @@
+{
+    "ServiceSettings": {
```

Comment:
Would we need to update this as new settings are added to the server ? is there an easier way to maintain this ?

Reply:
Yes, if needed as a baseline config but not that frequent. For special case or new setting, initial approach is to use `await apiUpdateConfig(newConfig)` into a particular test file and only update the default config whenever such new setting is applicable to majority of the test files.
For Cypress in the past, we maintain the default config in a separate repo but it's hard to maintain and sometimes having the "passed on local but not on CI".